### ISSUE TITLE
Consumer/subscription dies out without reconnect

### ISSUE THREAD

**Issue by vicstr** [vitorluis/python-mercure#2]
Just spent the last day trying to figure out why the Consumer thread dies out after roughly ~60s of runtime, 😅
this is the way it works and it needs to do a reconnect to the endpoint, involving a `Last-Event-ID`.
Some details here: [https://mercure.rocks/spec#reconciliation](url)

The Consumer thread uses an older library, **sseclient-py**,
which does no reconciliation, so you have to handle reconciliation yourself.

There is a newer **sseclient** library that does reconciliation automatically,
attaching last-event-id, restarting the connection when it ends.

**sseclient 0.0.27**
[https://github.com/btubbs/sseclient](url)

Basically the same that used in dunglas subscribe example:
[https://github.com/dunglas/mercure/blob/v0.10.4/examples/subscribe/python.py](url)

Is there a reason to keep using sseclient-py specifically at this moment?
I've made a fork to make a version with btubbs/sseclient.
Leaving this out here in case anybody else gets swamped.

Thanks!
